config/config.go

```diff
@@ -176,6 +178,8 @@ type GeneralNotifierConfig struct {
 	MinResultsNum                int
 	PollIntervalInSeconds        uint64
 	AutoSendIntervalInSeconds    uint64
+	ProxyFinalityCheck           bool
```

Comment:
might move the bool param to the bottom for memory alignment purposes. also for `ElrondConfig`

Reply:
done